frontend/src/components/common/FlexBox/FlexBox.tsx

```diff
@@ -53,6 +55,8 @@ const getGap = ({ gap, rowGap, columnGap }: Pick<FlexBoxProps, 'gap' | 'rowGap'
 
 const S = {
   FlexBox: styled.div<FlexBoxProps>`
+    width: ${({ width }) => (typeof width === 'number' ? `${width}rem` : width)};
```

Comment:
```suggestion
    width: ${({ width }) => (typeof width === 'number' ? `${width}rem` : width)};
```
여기서 width가 undefined일 때를 분기처리 해주세요